godtools/App/Features/Lesson/Lesson/LessonViewModel.swift

```diff
@@ -27,6 +29,18 @@ class LessonViewModel: MobileContentPagesViewModel {
         closeTapped()
     }
     
+    override func viewDidFinishLayout(window: UIViewController, safeArea: UIEdgeInsets) {
+        super.viewDidFinishLayout(window: window, safeArea: safeArea)
+        
+        incrementUserCounterUseCase.incrementUserCounter(for: .lessonOpen(tool: resource.id))
```

Comment:
Oh yeah this isn't even needed on the pageDidAppear() life cycle.  I didn't catch that these were opens.

Reply:
viewDidFinishLayout() works.